src/controllers/order.controller.js

```
@@ -63,14 +68,78 @@ export const createOrderController = async (req, res) => {
+          total += cantidad * precios.plato;
+          diasConPlato.add(item.dia);
+          break;
+
+        case 'extra':
+          const precioExtra = {
+            1: precios.postre,
+            2: precios.ensalada,
+            3: precios.proteina
+          }[item.item_id];
+
+          if (precioExtra) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)